### PR TITLE
Add a caption

### PR DESCRIPTION
**Pull request by arpan-lol**
Nothing much, I added the little "about" caption in the beginning of the README, changed `downloading` to `How to Download.` Just some little improvements to make it look professional  :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Astromine
 [![Staging Build](https://github.com/Chainmail-Studios/Astromine/workflows/Staging%20Build/badge.svg)](https://github.com/Chainmail-Studios/Astromine/actions?query=workflow%3A%22Staging+Build%22)
 
-Explore space; witness space; die in space.
+A technological space content mod for Fabric.
+Explore the space, witness the space, die in the space!
```
Comment (shedaniel) [inline README.md]: Can we not change this part, this is unnecessary

**Review by shedaniel — CHANGES_REQUESTED** [review]

**Review by arpan-lol — COMMENTED** [review]
okay

**Comment by shedaniel** [comment]
Please update the PR accordingly.

**Comment by arpan-lol** [comment]
I can't, it says I can't approve it myself. A moderator needs to do that.
Untitled

**Comment by shedaniel** [comment]
You are not supposed to review on the PR, edit the branch.

**Comment by arpan-lol** [comment]
there. Sorry for the inconvinience, I am not wuite much familiar with the pull request system.

**Review by shedaniel — APPROVED** [review]

**Pull request merged**